specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/costmanagement.exports.json

```diff
@@ -0,0 +1,799 @@
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Export"
+            },
+            "description": "Parameters supplied to the CreateOrUpdate Export operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/Export"
+            }
+          },
+          "201": {
+            "description": "Created.",
+            "schema": {
+              "$ref": "#/definitions/Export"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common-types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Exports"
+        ],
+        "operationId": "Exports_Delete",
+        "description": "The operation to delete a export.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/costmanagement/"
+        },
+        "x-ms-examples": {
+          "ExportDeleteBySubscription": {
+            "$ref": "./examples/ExportDeleteBySubscription.json"
+          },
+          "ExportDeleteByResourceGroup": {
+            "$ref": "./examples/ExportDeleteByResourceGroup.json"
+          },
+          "ExportDeleteByBillingAccount": {
+            "$ref": "./examples/ExportDeleteByBillingAccount.json"
+          },
+          "ExportDeleteByEnrollmentAccount": {
+            "$ref": "./examples/ExportDeleteByEnrollmentAccount.json"
+          },
+          "ExportDeleteByDepartment": {
+            "$ref": "./examples/ExportDeleteByDepartment.json"
+          },
+          "ExportDeleteByManagementGroup": {
+            "$ref": "./examples/ExportDeleteByManagementGroup.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/scopeExportParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/exportNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common-types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.CostManagement/exports/{exportName}/run": {
+      "post": {
+        "tags": [
+          "Exports"
+        ],
+        "operationId": "Exports_Execute",
+        "description": "The operation to execute an export.",
```

Comment:
Instead of calling it Exports_Execute, I'd suggest to call this something that sounds more like ExportX or RunExportJob, as the term 'execute' sounds overly format and/or with negative connotations.

Reply:
This caused a breaking change, updated other instances of "execute" than the ones that caused breaking changes.